README.md

```diff
@@ -282,11 +282,22 @@ encoding of the request. The parsing can be aborted by throwing an error.
 ## Errors
 
 The middlewares provided by this module create errors depending on the error
-condition during parsing. The errors will typically have a `status`/`statusCode`
-property that contains the suggested HTTP response code, an `expose` property
-to determine if the `message` property should be displayed to the client, a
-`type` property to determine the type of error without matching against the
-`message`, and a `body` property containing the read body, if available.
+condition during parsing. 
+
+This is passed to an Express error handling middleware, if one is defined, 
+in the usual way. Therefore, further inspection and handling can be performed 
+in another error handling middleware.
+
+The error provided is the original error thrown, with additional properties to 
```

Comment:
I'm not sure if this part actually clarifies anything. Like how would one know what "the original error thrown" is? Especially since most of the errors here are ever actually thrown at all internally?

Reply:
My intention here was to clarify that the error passed is isn't simply the additional properties, but also the actual Error object which contains name, message, stack, etc properties.

Unsure on how to phrase it better, but welcome to any suggestions or edits you have in mind.

Reply:
At least state that it is an Error object somewhere, otherwise it doesn't really say anything different than it did before, as it also said these were additional properties, not the only properties, prior to this change.